libs/block-mesh-manager/src/main.rs

```diff
@@ -83,8 +84,18 @@ async fn run() -> anyhow::Result<()> {
     let _ = create_test_user(&mut transaction).await;
     transaction.commit().await?;
 
-    let ws_connection_manager = ConnectionManager::new();
-    let _reports_cron_task = ws_connection_manager.cron_reports(Duration::from_secs(60)); // FIXME
+    let mut ws_connection_manager = ConnectionManager::new();
+    let _reports_cron_task = ws_connection_manager
+        .cron_reports(
+            Duration::from_secs(60),
+            vec![
+                WsServerMessage::RequestUptimeReport,
+                WsServerMessage::RequestBandwidthReport,
+            ],
+            100,
+            db_pool.clone(),
+        )
+        .await;
```

Comment:
Strange to call some inner function that spawns a loop.
Should check if we can extract to a top level function that runs with some arguments like other background tasks.
With this structure, it's as if the ConnectionManager owns it.